src/fontra/views/editor/panel-transformation.js

```diff
@@ -451,6 +530,77 @@ export default class TransformationPanel extends Panel {
     };
   }
 
+  async doPathOperations(pathOperationFunc, undoLabel) {
+    let { point: pointIndices } = parseSelection(this.sceneController.selection);
+    pointIndices = pointIndices || [];
+
+    const positionedGlyph =
+      this.sceneController.sceneModel.getSelectedPositionedGlyph();
+
+    const allContourIndices = [...range(positionedGlyph.glyph.path.numContours)];
+    const selectedContourIndices =
+      !pointIndices.length && undoLabel === "Remove overlaps"
```

Comment:
I don't like to switch things based on a label. You could compare `pathOperationFunc` with `unionPath` instead. But maybe do that once at the top of this function, and use a variable for that flag.

Reply:
Say `doUnion`